src/through.ml

```diff
@@ -0,0 +1,36 @@
+module Make (C : S.Cache) (DB : S.DB with type key = C.key) = struct
+  type 'a t = 'a C.t
+  type 'a d = 'a DB.d
+  type key = C.key
+
+  let v c = C.v c
+  let stats t = C.stats t
+  let is_empty t = C.is_empty t
+  let capacity t = C.capacity t
+  let size t = C.size t
+  let clear t = C.clear t
```

Comment:
I think that `Make` should implement `S.DB`. I don't see the point of implementing `S.Cache`. @pascutto

Reply:
need to discuss about that

Reply:
What about this then?
```
module type THROUGH = sig
  type 'a t
  type key

  val v : unit -> 'a t
  val mem : 'a t -> key -> bool
  val find : 'a t -> key -> 'a
  val replace : 'a t -> key -> 'a -> unit
  val remove : 'a t -> key -> unit

  module Cache : sig
    val stats : 'a t -> Stats.t
    val is_empty : 'a t -> bool
    val capacity : 'a t -> int
    val size : 'a t -> int
    val clear : 'a t -> unit
    val mem : 'a t -> key -> bool
    val find : 'a t -> key -> 'a
    val find_opt : 'a t -> key -> 'a option
    val remove : 'a t -> key -> unit
  end
end
```